compiler/vmdef.nim

```diff
@@ -93,6 +93,7 @@ type
     opcNGetType,
     opcNStrVal,
     opcNSigHash,
+    opcNSymBodyHash
```

Comment:
Instead of yet another VM opcode, can you instead use the VM's callback mechanism?

Reply:
done,  callback mechanism worked well